src/main/webapp/map.html

```diff
@@ -10,13 +10,22 @@
     <script>
       var map;
       var markers = [];
+      var count = 0;
+
+      function clearMarkers() {
+        for (var i = 0; i < markers.length; i++) {
+          markers[i].setMap(null);
```

Comment:
Everything looks good, but I don't see this method being called anywhere, so I'm wondering what it does?

Reply:
So I used to have a feature where you could drop the markers across the map when a button was pressed, but now that we have a lot more data points I didn't think it would make sense to drop all of them one by one. I can either re-implement that feature of dropping the markers or just get rid of the function entirely, what do you think?

Reply:
Honestly, I don't know. I think if it will take a while, it's probable better not to do that? But I'm also an impatient user for all things, so I'm biased. If some app makes me watch an animation before logging in, I stop using it XD